Homework11/DynamicPropertiesTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace Homework11
+{
+    public class DynamicPropertiesTests
+    {
+        private IWebDriver _driver;
+        private IJavaScriptExecutor _javascriptExecutor;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            _driver = new ChromeDriver();
+            _javascriptExecutor = (IJavaScriptExecutor)_driver;
+            _driver.Manage().Window.Maximize();
+        }
+
+        [SetUp]
+        public void SetUp()
+        {
+            _driver.Navigate().GoToUrl("https://demoqa.com/elements");
+            var navigationButton = _driver.FindElement(By.Id("item-8"));
+            ScrollToElement(navigationButton);
+            navigationButton.Click();
+        }
+
+        [Test]
+        public void WaitAndClickButtonTest()
+        {
+            // Wait for 5 seconds till the button appears
+            WebDriverWait wait = new WebDriverWait(_driver, TimeSpan.FromSeconds(5));
```

Comment:
The same comments:
WebDriverWait object for all class
using Until method to return IWebElement

Reply:
Fixed.